docs/scripting/script-net.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Script Requests API
+category: Tutorials
+tags:
+	- experimental
+---
+
+::: warning
+The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft beta 1.19.70
+:::
+
+::: warning
+This module can only be used on Bedrock Dedicated Server.
+:::
+
+In Scripting API, you can send and receive HTTP-based requests to interact with the internet. For more detailed infomation please visit [Microsoft docs](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server-net/minecraft-server-net).
+
+## Setup
+
+**Dependencies**
+
+Like other modules, you will need to add the dependency into your `manifest.json`
+
+```json
+{
+	"dependencies": [
+		{
+			"module_name": "@minecraft/server-net",
+			"version": "1.0.0-beta"
+		}
+	]
+}
+```
+
+# Enable module in Bedrock Dedicated Server
+
+1. Download Bedrock Server from https://www.minecraft.net/en-us/download/server/bedrock
```

Comment:
Download the Bedrock Server from the [Minecraft website](https://www.minecraft.net/en-us/download/server/bedrock)